packages/plexus/src/DirectedGraph/types.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
does the year here matter? github is of the impression that this is a new file (not that types.js was renamed types.tsx)

Reply:
@yurishkuro How do you feel about omitting the year from copyright notices? I think it's pretty common to leave it out?

https://github.com/facebook/jest/blob/51817fd79a3744078a82861f3763dc346ca0cbd6/packages/babel-jest/src/index.ts#L1-L10